python/main.py

```diff
@@ -47,20 +49,13 @@ def db_toList(items):
     return objects_list 
 
 def image_toHash(image):
```

Comment:
I guess it'll be better to rename the parameter since there is a constant called `image‘ defined in line 27. 
Btw, I guess it is jsut a small mistake from fetching upstream, but 
- you might want to delete line 26 since it's unrelated to directory structure. 
- in line 134 and 149, both `images` and `image` are used concurrently and you might want to unify into single directory (maybe `image`?).

Reply:
Thanks for the review.
I understand that 
- 〈hash〉.jpg is in `image` 
- 〈name〉.jpg is stored in `images`

I thought 
`get_image(〈hash〉.jpg)` in `main.py` looks for 〈hash〉.jpg in `image`, 
if not, default.jpg has to be taken out from `images`. 

So, I have left the `images` path in Line26 for this purpose.
Could you tell me if I'm wrong?